PlaceRouteHierFlow/placer/PlacerHyperparameters.h

```diff
@@ -13,6 +13,8 @@ class PlacerHyperparameters {
   double LAMBDA = 1.0;
 
   bool use_analytical_placer = false;
+  string placement_info_json_path = "";
+  bool use_external_placement_info = false;
```

Comment:
@stevenmburns It would be easier for me to do this in placer. Only need to specify the path to the json file in `placement_info_json_path` and set `use_external_placement_info` to true.

Reply:
Okay. I'm changing `placement_info_json_path` to `placement_info_json`. It will not be a path to a file, but instead will be a string which holds the contents of the file. (We don't necessarily what to create a file for this---passing a JSON string is better.)

Reply:
That is also fine.